misc/deployments/staging.gno.land/docker-compose.yml

```diff
@@ -1,130 +1,108 @@
-version: "2"
 
 services:
+  traefik:
+    image: "traefik:v2.11"
+    restart: unless-stopped
+    command:
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+      - "--entrypoints.rpc.address=:26657"
+      - "--entrypoints.web.http.redirections.entrypoint.to=websecure"
+      - "--entrypoints.web.http.redirections.entrypoint.scheme=https"
+      - "--entrypoints.web.http.redirections.entrypoint.permanent=true"
+      - "--entryPoints.web.forwardedHeaders.insecure"
+      - "--entrypoints.traefik.address=:8080"
+      - "--entrypoints.websecure.address=:443"
+
+      - "--certificatesresolvers.le.acme.tlschallenge=true"
+      - "--certificatesresolvers.le.acme.email=dev@gno.land"
+      - "--certificatesresolvers.le.acme.storage=/letsencrypt/acme.json"
+    ports:
+      - "80:80"
+      - "443:443"
+      - "26657:26657"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock:ro"
+      - ./letsencrypt:/letsencrypt
+
   gnoland:
-    container_name: gnoland
-    build: ../../..
+    image: ghcr.io/gnolang/gno/gnoland:master
+    restart: on-failure
+    entrypoint: /entrypoint.sh
+    working_dir: /gnoroot
     environment:
-      - VIRTUAL_HOST=rpc.staging.gno.land
-      - VIRTUAL_PORT=26657
-      - LETSENCRYPT_HOST=rpc.staging.gno.land
-      - LOG_LEVEL=4
-    working_dir: /opt/gno/src/gno.land
-    command:
-      - gnoland
-      - start
-      - --skip-failing-genesis-txs
-      - --chainid=staging
-      - --genesis-remote=staging.gno.land:26657
+      CHAIN_ID: staging
+      MONIKER: gno-staging
     volumes:
-      - "./data/gnoland:/opt/gno/src/gno.land/gnoland-data"
+      - ./gnoland.entrypoint.sh:/entrypoint.sh
     ports:
       - 26656:26656
-      - 26657:26657
-    restart: on-failure
-    logging:
-      driver: "json-file"
-      options:
-        max-file: "10"
-        max-size: "100m"
+      - 36657:26657
```

Comment:
I'm in favor of completely dropping this port, given as this testnet is basically unused